specifyweb/frontend/js_src/lib/components/QueryComboBox/index.tsx

```diff
@@ -352,6 +354,18 @@ export function QueryComboBox({
     hasTablePermission(field.relatedTable.name, 'create');
 
   const isReadOnly = React.useContext(ReadOnlyContext);
+
+  const viewButton = (
+    <DataEntry.View
+      aria-pressed={state.type === 'ViewResourceState'}
+      className="ml-1"
+      disabled={
+        formatted?.resource === undefined ||
+        collectionRelationships === undefined
+      }
+      onClick={(): void => handleOpenRelated(true)}
+    />
+  );
```

Comment:
```suggestion
  const viewButton = formatted?.resource === undefined ||
          hasTablePermission(formatted.resource.specifyTable.name, 'read') ? (
    <DataEntry.View
      aria-pressed={state.type === 'ViewResourceState'}
      className="ml-1"
      disabled={
        formatted?.resource === undefined ||
        collectionRelationships === undefined
      }
      onClick={(): void => handleOpenRelated(true)}
    />
  ) : undefined;
```

then you don't need to duplicate the condition in both places